doc/tutorials/report.rst

```diff
@@ -56,6 +56,12 @@ To generate the report in parallel::
     mne report --path MNE-sample-data/ --info MNE-sample-data/MEG/sample/sample_audvis-ave.fif \ 
         --subject sample --subjects-dir MNE-sample-data/subjects --verbose --jobs 6
 
+The report rendered on sample-data is shown below:
+
+
+    .. image:: mne-report.png
```

Comment:
not by much. Unless I resize. I tried using a weblink and that too doesn't seem to work :\